extension/test/e2e/klarna.spec.js

```diff
@@ -20,7 +20,7 @@ const {
 } = require('../../src/config/constants')
 
 // Flow description: https://docs.adyen.com/payment-methods/klarna/web-component#page-introduction
-describe('::klarnaPayment::', () => {
+describe.skip('::klarnaPayment::', () => {
```

Comment:
Please merge from master.

Reply:
IBAN is suddenly not required in Klarna e2e. I just update it here. I will apply the change in other pending PRs later.
https://github.com/commercetools/commercetools-adyen-integration/pull/769/commits/533ac5f0ea8679432a36d9b687aee7fd42f69d65